### ISSUE TITLE
feat: onClick 'mouseButton' parameter to detect what mouseButton was pressed

### ISSUE THREAD

**Issue by lajbel** [marklovers/kaplay#6]

**Comment by mflerackers** [comment]
I think using [isMousePressed](https://kaboomjs.com/#isMousePressed)(button?: MouseButton) you can check whether the click was triggered using a certain button. But this may be more cumbersome.

**Comment by lajbel** [comment]
I think it changes the style of the user, passing from `onClick()` to `onUpdate(() => { if(isMousePressed) }), so we should have parity between event and if functions

**Comment by lajbel** [comment]
If version:

```js
onUpdate(() => {
    if(isMousePressed("right")) {
        console.log("right");
    }
});
```

event version:
```js
onClick((btn) => {
    console.log(btn); // right
});
```

**Comment by lajbel** [comment]
fixed on: 5e16da1
